docs/source/methods/index.rst

```diff
@@ -0,0 +1,16 @@
+*************
+Methods
+*************
+
+.. list-table::
+    :widths: 20 80
+
+    * - :doc:`CECM <../CECM>`
```

Comment:
I would use the same heading as in the notebooks: Fundamental Diagrams at Measurement Lines. Then it is consistent and more understandable. CECM was a working name, not sure if somebody knows what it stands for.